src/adapters/bidderFactory.js

```diff
@@ -523,10 +523,9 @@ export const processBidderRequests = hook('sync', function (spec, bids, bidderRe
 export const registerSyncInner = hook('async', function(spec, responses, gdprConsent, uspConsent, gppConsent) {
   const aliasSyncEnabled = config.getConfig('userSync.aliasSyncEnabled');
   if (spec.getUserSyncs && (aliasSyncEnabled || !adapterManager.aliasRegistry[spec.code])) {
-    let filterConfig = config.getConfig('userSync.filterSettings');
     let syncs = spec.getUserSyncs({
-      iframeEnabled: !!(filterConfig && (filterConfig.iframe || filterConfig.all)),
-      pixelEnabled: !!(filterConfig && (filterConfig.image || filterConfig.all)),
+      iframeEnabled: isActivityAllowed(ACTIVITY_SYNC_USER, activityParams(MODULE_TYPE_BIDDER, spec.code, { syncType: 'iframe' })),
```

Comment:
this would log a warning when denied, but the bidder is not necessarily trying to use the wrong type of sync - we are just telling them whether image or iframes are allowed. Internally the activity is checked using [`canBidderRegisterSync`](https://github.com/prebid/Prebid.js/blob/5e57caa8ceb422955725976740d79a24ca6a7770/src/userSync.js#L331), which on its own does not generate warnings.

Reply:
HI @dgirardi - I'll update source branch (https://github.com/grupawp/Prebid.js/tree/fix-syncoptions) with Your changes